scripts/processors/jartic_processor.py

```
@@ -123,12 +123,14 @@ class JARTICProcessor(BaseProcessor):
     
     def __init__(self, country: str = 'JP', data_dir: Optional[Path] = None, 
                  max_rows: Optional[int] = None, n_workers: Optional[int] = None,
-                 input_file: Optional[str] = None, enable_checkpoints: bool = True):
+                 input_file: Optional[str] = None, enable_checkpoints: bool = True,
```

Ruff (0.12.2)

Boolean-typed positional argument in function definition

(FBT001)